july1st2019/july1st2019_model.nlogo

```diff
@@ -1313,7 +1313,7 @@ INPUTBOX
 197
 259
 Simulation-time-min
-600.0
+480.0
```

Comment:
These parameters should be specified as parameters in the experiment XML.  Changing these here will only the GUI version which has nothing to do with headless mode.  I will make a new PR to incorporate these changes.